test/performance/.benchmarks/0001_performance.json

```diff
@@ -0,0 +1,667 @@
+                "iqr_outliers": 0,
+                "stddev_outliers": 6,
+                "outliers": "6;0",
+                "ld15iqr": 3.45042175399999,
+                "hd15iqr": 3.519597754000017,
+                "ops": 0.2872890138261435,
+                "total": 52.21222976899992,
+                "data": [
+                    3.4835063849999983,
+                    3.516568006,
+                    3.4883376699999644,
+                    3.518792517999998,
+                    3.4697337310000194,
+                    3.499725911999974,
+                    3.467015816000014,
+                    3.46768780299999,
+                    3.519597754000017,
+                    3.472595393000006,
+                    3.4619791520000263,
+                    3.487405907999971,
+                    3.452087153999969,
+                    3.45042175399999,
+                    3.4567748129999814
+                ],
+                "iterations": 1
+            }
+        },
+        {
+            "group": null,
+            "name": "test_import_methods[file]",
+            "fullname": "test/performance/connection/http_transport_import_test.py::test_import_methods[file]",
+            "params": {
+                "import_method": "import_from_file",
+                "data_creator": "create_csv"
+            },
+            "param": "file",
+            "extra_info": {},
+            "options": {
+                "disable_gc": false,
+                "timer": "perf_counter",
+                "min_rounds": 5,
+                "max_time": 1.0,
+                "min_time": 5e-06,
+                "warmup": false
+            },
+            "stats": {
+                "min": 1.3742191810000577,
+                "max": 7.116984646999981,
+                "mean": 2.353781834733339,
+                "stddev": 1.3511345104802155,
+                "rounds": 15,
+                "median": 2.053984645000014,
+                "iqr": 0.15911088400005724,
+                "q1": 1.997610069999979,
+                "q3": 2.1567209540000363,
+                "iqr_outliers": 4,
+                "stddev_outliers": 1,
+                "outliers": "1;4",
+                "ld15iqr": 1.9771298090000755,
+                "hd15iqr": 2.5519972299999836,
+                "ops": 0.42484821033266684,
+                "total": 35.306727521000084,
+                "data": [
+                    1.3742191810000577,
+                    2.0218934409999747,
+                    2.1148670270000594,
+                    1.9771298090000755,
+                    2.0785514680000006,
+                    2.053984645000014,
+                    2.073814807999952,
+                    2.037034587999983,
+                    1.9895156129999805,
+                    2.0482951599999524,
+                    1.3755199900000434,
+                    2.322247650999998,
+                    2.1706722630000286,
+                    7.116984646999981,
```

Comment:
The imports outlier's don't look random, mostly the round before the last is slow. Maybe we force a flush to disk of the uncommitted data at this point, because we don't cleanup.